rest-api/tools/ehr_upload.py

```diff
@@ -1,97 +1,132 @@
-        session, args.organization, args.source_bucket, args.destination_bucket)
-
-def get_participants_under_sites(session, organization, source_bucket, destination_bucket):
-  sql = """
-  select p.participant_id, google_group from participant p left join site s on
-  p.site_id = s.site_id
-  left join participant_summary summary on p.participant_id = summary.participant_id
-  where p.organization_id in (select organization_id from organization where external_id = '{}')
-  and summary.consent_for_electronic_health_records = 1
-  and summary.consent_for_study_enrollment = 1;
-   """.format(organization)
-  cursor = session.execute(text(sql))
-  try:
-    results = cursor.fetchall()
-    if results:
-      results = [(int(i), str(k)) for i, k in results]
-      for participant, google_group in results:
-        gsutil = "gsutil -m cp gs://" + source_bucket + "/Participant/P" + str(
-          participant) + "/* " + "gs://" + destination_bucket + "/Participant/" + \
-                 google_group + "/P" + str(participant) + "/"
-
-        gsutil = shlex.split(str(gsutil))
-        system_call = subprocess.Popen(gsutil)
-        system_call.communicate()[0]
-        if system_call.returncode == 0:
-          print "Successfully moved folder " + google_group + '/P' + str(participant)
-        else:
-          print "There was an error moving folder " + google_group + '/P' + str(participant)
-          print "return code is : " + str(system_call.returncode)
-    else:
-      print "No participants paired with sites found for organization: " + organization
-  finally:
-    cursor.close()
+SOURCE_BUCKET = 'ptc-uploads-all-of-us-rdr-prod'
+BUCKET_NAME = 'Bucket Name'
+AGGREGATING_ORG_ID = 'Aggregating Org ID'
+ORG_ID = 'Org ID'
+ORG_STATUS = 'Org Status'
 
 
-def get_participants_without_site_pairing(session, organization, source_bucket, destination_bucket):
-  sql = """
-    select p.participant_id from participant p
+def get_sql(organization):
+  site_pairing_sql = """
+    select p.participant_id, google_group from participant p left join site s on
+    p.site_id = s.site_id
     left join participant_summary summary on p.participant_id = summary.participant_id
-    where p.site_id is NULL and p.organization_id in (
-    select organization_id from organization where external_id = '{}')
+    where p.organization_id in (select organization_id from organization where external_id = '{}')
+    and s.google_group is not null
     and summary.consent_for_electronic_health_records = 1
     and summary.consent_for_study_enrollment = 1;
-   """.format(organization)
-  cursor = session.execute(text(sql))
-  try:
+     """.format(organization)
+
+  no_site_pairing_sql = """
+      select p.participant_id from participant p
+      left join participant_summary summary on p.participant_id = summary.participant_id
+      where p.site_id is NULL and p.organization_id in (
+      select organization_id from organization where external_id = '{}')
```

Comment:
Per @robabram 's comment, I agree it would be better practice to use a SQL parameter here rather than string building. Understood that the value comes from a flag, but it looks like a safe and low effort fix